src/steamship/agents/llms/steamship_llm.py

```diff
@@ -0,0 +1,110 @@
+import logging
+from pprint import pformat
+from typing import List, Optional
+
+from steamship import Block, File, MimeTypes, PluginInstance, Steamship, SteamshipError, Tag
+from steamship.agents.schema import Tool
+from steamship.data import GenerationTag, TagKind
+from steamship.plugin.capabilities import (
+    CapabilityPluginRequest,
+    CapabilityPluginResponse,
+    ConversationSupport,
+    FunctionCallingSupport,
+    SystemPromptSupport,
+)
+
+
+class SteamshipLLM:
+    """A class wrapping LLM plugins."""
+
+    def __init__(self, plugin_instance: PluginInstance):
+        # TODO (PR callout): In the spirit of getting folks to pin to specific instances more often, take the route of
+        #  using specific PluginInstances rather than client.use_plugin all the time with the params that we determine?
+        self.client = plugin_instance.client
+        self.plugin_instance = plugin_instance
+
+    @staticmethod
+    def with_gpt4(client: Steamship, temperature: float = 0.4, max_tokens: int = 256):
```

Comment:
I feel like we've been burned before by not allowing `kwargs` on the end of these types of methods. I suggest allowing the `config` to include `**kwargs` style expansion for forward-compat with the plugin.